src/components/form/form.tsx

```diff
@@ -74,6 +74,12 @@ export class Form {
     @Prop()
     public propsFactory?: (schema: Record<string, any>) => Record<string, any>;
 
+    /**
+     * Custom function to customize the default error messages provided by JSON Schema
+     */
+    @Prop()
+    public transformErrors?: (errors: AjvError[]) => AjvError[];
```

Comment:
We don't want to expose types from `rjsf` to our consumers, as that locks us into `rjsf`, it also means that updating `rjsf` could become more difficult if there are breaking changes made to the `AjvError` interface. Can we create a proxy interface, that exposes reasonably generic properties, likely to still be available (or made available through an adapter) if the underlying implementation changes or is swapped out?

Reply:
I've made some changes based on your comment

Reply:
Nice! I'll have a look 👍